src/Fantomas.Core.Tests/FunctionDefinitionTests.fs

```diff
@@ -111,43 +111,31 @@ let ``should keep identifiers with + in double backticks`` () =
 """
 
 [<Test>]
-let ``double backticks with non-alphanum character, 776`` () =
+[<TestCase "let ``!foo hoo`` () = ()">]
```

Comment:
I really dislike that you always seem to touch a bit more code than you absolutely need to.
You could have left this test alone.

Reply:
Nah, keep it in. It is fine. Just keep it in mind in the future.

Reply:
👍🏻 I know I can have a tendency to change more than necessary. Usually it's a result of tinkering around to figure out a problem (as in this case). I try to clean up anything that didn't end up actually being necessary as I fix things but I'm sorry that I can miss things on occasion. I'm always happy to trim things down